com.github.fabrialberio.pinapp.yml

```diff
@@ -0,0 +1,40 @@
+id: com.github.fabrialberio.pinapp
+sdk: org.gnome.Sdk
+runtime: org.gnome.Platform
+runtime-version: "43"
+command: pinapp
+
+finish-args:
+  - --share=ipc
+  - --device=dri
+  - --socket=fallback-x11
+  - --socket=wayland
+
+  - --env=XDG_DATA_DIRS=/var/lib/flatpak/exports/share:/app/share:/usr/share:/usr/share/runtime/share:/run/host/user-share:/run/host/share
```

Comment:
can you explain this?

Reply:
A lot of the folders included in XDG_DATA_DIRS are unnecessary, cleaned up in latest commit
Still, that is the best way I found to get flatpak application icons to display in the application, because for some reason declaring the folders explicitly (for example `--filesystem=/var/lib/flatpak/exports/share/icons:ro`) doesn't work.